.pep8

```diff
@@ -0,0 +1,2 @@
+[pep8]
```

Comment:
Let's remove this one (shouldn't be a part of the pull request as they don't use them)